dev/io.openliberty.mcp.internal/test/io/openliberty/mcp/internal/test/MCPServerToolsListTest.java

```diff
@@ -40,75 +39,78 @@ public void setup() throws Exception {
-                                          "var5": {
-                                              "description": "short -> number",
-                                              "type": "number"
-                                          }
-                                      },
-                                      "required": [
-                                          "var1",
-                                          "var2",
-                                          "var3",
-                                          "var4",
-                                          "var5"
-                                      ]
-                                  }
-                              }
-                          ]
+                            {
+                                "description": "Checks if primitives arguments are handled by the Json Serialiser",
+                                "inputSchema": {
+                                    "type": "object",
+                                    "properties": {
+                                        "var5": {
+                                            "type": "integer",
+                                            "description": "short -> number"
+                                        },
+                                        "var4": {
+                                            "type": "number",
+                                            "description": "float -> number"
+                                        },
+                                        "var3": {
+                                            "type": "integer",
+                                            "description": "byte -> number"
+                                        },
+                                        "var2": {
+                                            "type": "number",
+                                            "description": "double -> number"
+                                        },
+                                        "var1": {
+                                            "type": "integer",
+                                            "description": "long -> number"
+                                        }
+                                    },
+                                    "required": [
+                                        "var5",
+                                        "var4",
+                                        "var3",
+                                        "var2",
+                                        "var1"
+                                    ]
+                                },
+                                "name": "parseAllPrimitiveNumbers",
+                                "title": "parse All PrimitiveNumbers"
+                            }
+                        ]
 
-                                                                  """;
+                                                              """;
```

Comment:
Doing the change like this, though likely correct, makes it very hard to see what actually changed during the review.